src/cli/onefuzz/status/cache.py

```diff
@@ -4,39 +4,75 @@
 # Licensed under the MIT License.
 
 import logging
-from datetime import datetime, timedelta, timezone
+from datetime import datetime
 from enum import Enum
 from typing import Any, Dict, List, Optional, Set, Tuple, Union
 from uuid import UUID
 
-from onefuzztypes.enums import ContainerType, JobState, NodeState, PoolState, TaskState
-from onefuzztypes.models import Job, Node, Pool, Task
+from onefuzztypes.enums import ContainerType, JobState, NodeState, TaskState, TaskType
+from onefuzztypes.events import (
+    EventCrashReported,
+    EventFileAdded,
+    EventJobCreated,
+    EventJobStopped,
+    EventMessage,
+    EventNodeCreated,
+    EventNodeDeleted,
+    EventNodeStateUpdated,
+    EventPoolCreated,
+    EventPoolDeleted,
+    EventTaskCreated,
+    EventTaskFailed,
+    EventTaskStateUpdated,
+    EventTaskStopped,
+    EventType,
+)
+from onefuzztypes.models import (
+    Job,
+    JobConfig,
+    Node,
+    Pool,
+    Task,
+    TaskContainers,
+    UserInfo,
+)
+from onefuzztypes.primitives import Container
 from pydantic import BaseModel
 
-MESSAGE = Tuple[datetime, str, str]
+MESSAGE = Tuple[datetime, EventType, str]
 
 MINUTES = 60
 HOURS = 60 * MINUTES
 DAYS = 24 * HOURS
 
 
-def fmt_delta(data: timedelta) -> str:
-    result = []
+class MiniNode(BaseModel):
```

Comment:
What are these `Mini*` classes?

Reply:
We don't send the full context of the Node/Task/Job, etc via SignalR.  As such, these are slimmed down representations of the models specifically for the context of `status top`.

Added a comment to each of them to describe that.